packages/ui-kit/src/utils/lord-icon.ts

```diff
@@ -0,0 +1,10 @@
+import Suitcase from '../icons/generated/Suitcase';
+
+export const getStaticSvgComponent = (iconName: string) => {
+  switch (iconName) {
```

Comment:
could be an object with props that are the name keys.  Would maybe have better Typing support by TS (the object would have an implicit type) and easier to maintain maybe.